### ISSUE TITLE
[bug] jest test error

### ISSUE THREAD

**Issue by vpachedzhi** [danpacho/tailwindest#49]
**Describe the bug**
When running tests with `jest` and `ts-jest` the following error was observed

```
export { createTools };
    ^^^^^^
    SyntaxError: Unexpected token 'export'
    > 1 | import { createTools, type Tailwindest } from 'tailwindest';
```

**Expected behavior**
Tests should run normally without the error above

**Additional context**
```typescript
// jest.config.ts
module.exports = {
  testEnvironment: 'jsdom',
  transform: {
    '^.+\\.(ts|tsx|js|jsx)$': 'ts-jest',
  },
  moduleNameMapper: {
    "^@/(.*)": "<rootDir>/src/$1",
  },
  transformIgnorePatterns: [
    "node_modules/(?!tailwindest/)", // fixed the issue
  ]
};
```
`tsconfig.json`
```json
{
  "compilerOptions": {
    "target": "es5",
    "lib": ["dom", "dom.iterable", "esnext"],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": true,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "react-jsx",
    "incremental": true,
    "paths": {
      "@/*": ["./src/*"]
    }
  },
  "types": ["node", "jest", "@testing-library/jest-dom"],
  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
  "exclude": ["node_modules"]
}
```
`"ts-jest": "^29.1.0"` and `"jest": "^29.5.0",`

**Comment by danpacho** [comment]
Can you show me test code that bug was occurred? That would be helpful for fixing the bug :)

**Comment by danpacho** [comment]
fixed via commonJs support at `v2.1.2`
